lib/modules/manager/sbt/extract.spec.ts

```diff
@@ -438,4 +441,68 @@ describe('modules/manager/sbt/extract', () => {
       ).toBeNull();
     });
   });
+
+  describe('extractAllPackageFiles()', () => {
```

Comment:
As this will take precedence, also add some checks for resulting deps and not only their registry urls